ci-operator/step-registry/ipi/conf/vsphere/windows-machineset/ipi-conf-vsphere-windows-machineset-ref.yaml

```diff
@@ -15,7 +15,7 @@ ref:
     - name: WINDOWS_NODE_REPLICAS
       default: "1"
     - name: WINDOWS_OS_ID
-      default: "windows-golden-images/windows-server-2022-template-qe"
+      default: "windows-server-2022-template-qe-20241104"
```

Comment:
I was thinking this image not under folder can get pruned

Reply:
I'll fix this. It's actually under windows-golden-images/